CHANGELOG.md

```diff
@@ -0,0 +1,62 @@
+# Changelog
```

Comment:
I think we could start a new changelog from now on, no need for the past entries, which are still part of the previous repo.